From 45a4324db9ffc5570d2507e9a86389e512c86c8a Mon Sep 17 00:00:00 2001
From: Ioana Alexandru <aioana@google.com>
Date: Fri, 26 Jul 2024 17:30:50 +0000
Subject: [PATCH 2/5] Remove visitPersonUri flagged code

Flag: EXEMPT removing code for rolled out flag
Bug: 281044385
Test: builds

(cherry picked from commit 0f4bf9fa13f6b5760c8b80103371db82f69813f8)
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:2a41d41657b94b81cb7245eff90b7e4824960c29)
Merged-In: I05ccd19d997063873d9df0f7c4e1906bd3f0e6f5
Change-Id: I05ccd19d997063873d9df0f7c4e1906bd3f0e6f5
---
 core/java/android/app/Person.java | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/core/java/android/app/Person.java b/core/java/android/app/Person.java
index 96f6f4eac372..c7432c571e43 100644
--- a/core/java/android/app/Person.java
+++ b/core/java/android/app/Person.java
@@ -189,10 +189,8 @@ public final class Person implements Parcelable {
      */
     public void visitUris(@NonNull Consumer<Uri> visitor) {
         visitor.accept(getIconUri());
-        if (Flags.visitPersonUri()) {
-            if (mUri != null && !mUri.isEmpty()) {
-                visitor.accept(Uri.parse(mUri));
-            }
+        if (mUri != null && !mUri.isEmpty()) {
+            visitor.accept(Uri.parse(mUri));
         }
     }
 
-- 
2.34.1

